interface-definitions/firewall.xml.in

```diff
@@ -571,6 +591,26 @@
                 </children>
               </node>
               #include <include/firewall/common-rule.xml.i>
+              <leafNode name="ip-length">
```

Comment:
As this node is used twice, please move it to a dedicated `.xml.i` file and then `#include <>` it.

Reply:
It's almost the same, but differs on the  help information, which is slightly different for both protocols.

Reply:
`<help>Packet size in bytes, including header and data</help>` will match for both IPv4 and IPv6 as the main information is "including header and data".